.storybook/editor-theme.js

```diff
@@ -0,0 +1,51 @@
+export const editorTheme = {
```

Comment:
small correction.
we have a naming convention in the eslint.
here is the rule.
```
    'unicorn/filename-case': [
      'error',
      {
        cases: {
          camelCase: true,
          pascalCase: true,
        },
      },
    ],
 ```
it's means all javascript or TS files should be `camelCase ` or `pascalCase`
Maybe it's not working don't the `dot` folders.
It needs to check why.
If you have time are welcome.

Reply:
thanks ! good job. only one correction